components/connect/Carousel.tsx

```diff
@@ -0,0 +1,149 @@
+import "react-responsive-carousel/lib/styles/carousel.min.css";
+
+import React, { useState, useEffect } from "react";
+import { Box, Flex, SimpleGrid } from "@chakra-ui/react";
+import { Carousel as ReactResponsiveCarousel } from "react-responsive-carousel";
+import { PlaygroundMenu } from "./PlaygroundMenu";
+import { ChakraNextImage } from "components/Image";
+import { useTrack } from "hooks/analytics/useTrack";
+
+const showcaseMenus = [
+  {
+    id: "pay-icon",
+    title: "Pay",
+    description:
+      "[Short desc - onboard users with fiat or crypto at any web3 point of sale]",
+    image: require("public/assets/product-pages/connect/icon-pay.png"),
+  },
+  {
+    id: "connect-icon",
+    title: "Connect",
+    description:
+      "[Short desc - onboard users with fiat or crypto at any web3 point of sale]",
+    image: require("public/assets/product-pages/connect/icon-connect.png"),
+  },
+  {
+    id: "account-abstraction-icon",
+    title: "Account Abstraction",
+    description:
+      "[Short desc - onboard users with fiat or crypto at any web3 point of sale]",
+    image: require("public/assets/product-pages/connect/icon-aa.png"),
+  },
+];
+
+const showcaseImages = [
+  require("public/assets/product-pages/connect/pay.png"),
+  require("public/assets/product-pages/connect/connect.png"),
+  require("public/assets/product-pages/connect/account-abstraction.png"),
+];
+
+const Carousel = ({ TRACKING_CATEGORY }: { TRACKING_CATEGORY: string }) => {
+  const [selectedShowCaseIdx, setSelectedShowCaseIdx] = useState(0);
+  const [hoveredCard, setHoveredCard] = useState(false);
+  const trackEvent = useTrack();
+
+  const increment = () => {
+    setSelectedShowCaseIdx((idx) => (idx === 2 ? 0 : idx + 1));
+  };
+
+  useEffect(() => {
+    if (!hoveredCard) {
+      const timer = setInterval(increment, 3500);
+
+      return () => clearInterval(timer);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why do we need this? can we avoid this somehow please?

Reply:
Yep